content/03.chemoproteomics.md

```diff
@@ -1,3 +1,44 @@
-## 2. Selective probing of membrane proteins with chemoproteomics {.page_break_before}
```

Comment:
I like the introduction and the summary of recent results. In the intro, I would make a better connection to membrane proteins, otherwise the transition is too abrupt. You could also add one summarizing sentence at the end.